src/parseTools.js

```diff
@@ -56,8 +55,8 @@ function preprocess(text, filenameHint) {
   const showStack = [];
   const showCurrentLine = () => showStack.every((x) => x == SHOW);
 
-  currentlyParsedFilename = filenameHint;
-  const fileExt = (filenameHint) ? filenameHint.split('.').pop().toLowerCase() : '';
+  currentlyParsedFilename = filename;
```

Comment:
Where is `currentlyParsedFilename` used / is it still necessary?

Reply:
I think this is global which is used elsewhere.. we may want to try to remove it, but I don't think its related to this change.

Reply:
Yeah it's not, just noticed here.